client/rest-high-level/src/main/java/org/elasticsearch/client/transform/transforms/TransformConfig.java

```diff
@@ -109,26 +109,26 @@ private static SyncConfig parseSyncConfig(XContentParser parser) throws IOExcept
     }
 
 
-    public static DataFrameTransformConfig fromXContent(final XContentParser parser) {
+    public static TransformConfig fromXContent(final XContentParser parser) {
         return PARSER.apply(parser, null);
     }
 
     /**
-     * Helper method for previewing a data frame transform configuration
+     * Helper method for previewing a transform configuration
      *
-     * The DataFrameTransformConfig returned from this method should only be used for previewing the resulting data.
+     * The TransformConfig returned from this method should only be used for previewing the resulting data.
      *
-     * A new, valid, DataFrameTransformConfig with an appropriate destination and ID will have to be constructed to create
+     * A new, valid, TransformConfig with an appropriate destination and ID will have to be constructed to create
      * the transform.
      * @param source Source configuration for gathering the data
      * @param pivotConfig Pivot config to preview
-     * @return A DataFrameTransformConfig to preview, NOTE it will have a {@code null} id, destination and index.
+     * @return A TransformConfig to preview, NOTE it will have a {@code null} id, destination and index.
      */
-    public static DataFrameTransformConfig forPreview(final SourceConfig source, final PivotConfig pivotConfig) {
-        return new DataFrameTransformConfig(null, source, null, null, null, pivotConfig, null, null, null);
+    public static TransformConfig forPreview(final SourceConfig source, final PivotConfig pivotConfig) {
+        return new TransformConfig(null, source, null, null, null, pivotConfig, null, null, null);
     }
 
-    DataFrameTransformConfig(final String id,
+    TransformConfig(final String id,
```

Comment:
indention for all these params...I cannot provide the suggestion to LOC outside of the change :(((((